lib/src/linter.dart

```diff
@@ -101,7 +101,7 @@ class Group implements Comparable<Group> {
           'https://www.dartlang.org/articles/style-guide/'));
 
   /// List of builtin groups in presentation order.
-  static Iterable<Group> get builtin => [errors, style, pub];
+  static const Iterable<Group> builtin = const [errors, style, pub];
```

Comment:
Good point.  This is internal though.  Moreover, `List`s can be modified and this should not be.

FWIW: this is a temporary shim anyway.  Odds are good that groups will be defined via an extension point when we have the rest of the plugin story plumbed in.